include/z64.h

```diff
@@ -969,6 +969,77 @@ typedef struct {
     /* 0x4C */ u32 unk_4C;
 } PreRender; // size = 0x50
 
+#define TRANS_TRIGGER_OUT -20 // transition is going out
+#define TRANS_TRIGGER_OFF 0 // transition is not active
+#define TRANS_TRIGGER_IN 20 // activate transition to come in
```

Comment:
I agree on the distinction being confusing, but couldnt figure out how to address it. I went with the fadeOut and fadeIn distinction made by the entrance table 

the variable means way more to the transition than the actual load that occurs after the transition is done, which is why i found it more appropriate to name it after the transition

Reply:
So if I get it right these mean:
```suggestion
#define TRANS_TRIGGER_OUT -20 // activate fading out transition (when entering scene)
#define TRANS_TRIGGER_OFF 0 // transition is not active
#define TRANS_TRIGGER_IN 20 // activate fading in transition (leads to loading next scene)
```
is that right, would that be better?

Reply:
If it's fine to name these after the fact they are used when changing scene, maybe `trigger_arrive` and `trigger_exit` would be more straightforward but idk.

That said, I'm fine with the current in/out refering to the visual transition instead of the scene transition, it's going to be kind of weird to wrap your head around it regardless, so as long as there are clear comments it's enough for me

So your call :) or if anyone has an opinion

Reply:
arrive and exit do make it easier to wrap your head around when just reading the defines. I do think these variables mean more for the transistion itself than the scene triggering part, but they are closely tied together so its probably fine. more opinions would be good

Reply:
This is one of the first things I looked at in this PR, and I did wonder what these meant with IN/OUT, so I think I'd prefer if they're associated to exit/enter instead.

Reply:
Dragorn's comments have made me realized that that the names are backwards to what the transitions are doing; `TRANS_TRIGGER_IN` initializes a fade out (fade to black/white/etc.) while `TRANS_TRIGGER_OUT` initializes a fade in (fade from black/white/etc. to the game).

I agree with fig in that the variables appear to be specific to the transition system; I almost feel like the weird values assigned might have been related to fade duration (fast fade black/white use a 20 updates/1 second? duration) at one point. But more to my point, I think we should name these variables based on what they do, not the coupling they have with initializing game state switches, and especially not directly tie them into scene switching since there are instances where you don't switch into a scene.

Reply:
I wouldnt say its wrong, but it depends on perspective. the current names are based off of the fade itself (coming) in, for the fade (going) out. That perspective difference is why the current names are not good, cause you can see it either way